test/Reader/ReaderTest.php

```diff
@@ -158,7 +158,7 @@ public function testImportsUriAndThrowsExceptionIfNotAFeed()
             $this->markTestSkipped('testImportsUri() requires a network connection');
         }
 
-        Reader\Reader::import('http://twitter.com/alganet');
+        Reader\Reader::import('http://example.com');
```

Comment:
Troubles with redirect to HTTPS protocol. I didn't want to deal with that https://travis-ci.org/zendframework/zend-feed/jobs/92411760#L317